docs/endpoints/post-token-refresh.md

```diff
@@ -31,6 +30,8 @@ Here's what you need to know about this endpoint:
 
 >NOTE: The integration environment and the production environment require different [API keys](../ref-info/glossary-uid.md#gl-api-key).
 
+{**GWH/SW_03 query. It doesn't say, anywhere, how the actual token is conveyed. Path parameter? Body? This ties in with my question #4.**}
```

Comment:
In the `POST '{environment}/v2/token/refresh'`  on line 17 above, you can just say add the content of the refresh_token value from previously token/gen or token/refresh response as the _POST body_ and that's how you call token/refresh

Reply:
Done. Though, would LOVE an example of that.